perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -0,0 +1,150 @@
+
+final_time_np=$((scale_w_np_end_time - scale_w_np_start_time))
+echo "execution time with network policy was $final_time_np s."
+
+echo "=========================================================================="
+echo ""
+
+# #DELETE NETWORK POLICY
+echo "Deleting network policy..."
+oc get networkpolicy -n ${NAMESPACE} --no-headers | awk {'print $1'} | xargs oc delete networkpolicy -n ${NAMESPACE}
+echo ""
+
+# SCALE DOWN DEPLOYMENT
+echo "Scaling down the deployment..."
+
+scale_down=$(oc get deployment $deployment_name --no-headers -n ${NAMESPACE} | awk -F ' {1,}' '{print $4}')
+echo ""
+
+oc scale --replicas $pod_scale_min deployment $deployment_name -n ${NAMESPACE}
+
+oc get deployment $deployment_name  -n ${NAMESPACE}
+
+check_deployment_pod_scale ${NAMESPACE} $deployment_name $scale_down $pod_scale_min
+echo ""
+
+echo "Phew! Let me catch my breath for a second..."
+sleep 5
+
+echo "=========================================================================="
+echo ""
+
+
+# SCALE UP DEPLOYMENT
+echo "Starting scale-up test without network policy..."
+
+scale_up_no_np=$(oc get deployment $deployment_name --no-headers -n ${NAMESPACE} | awk -F ' {1,}' '{print $4}')
+oc scale --replicas $pod_scale_max deployment $deployment_name -n ${NAMESPACE}; start_time=`date +%s`
+
+echo "start time: $start_time"
+check_deployment_pod_scale ${NAMESPACE} $deployment_name $scale_up_no_np $pod_scale_max
+
+end_time=`date +%s`
+echo "end time: $end_time"
+
+final_time_no_np=$((end_time - start_time))
+echo "execution time without network policy was $final_time_no_np s."
+
+echo "=========================================================================="
+echo ""
+
+echo "execution time with network policy was $final_time_np s."
+echo "execution time without network policy was $final_time_no_np s."
+
+
+
+if [[ ( $final_time_np -ge $final_time_no_np ) ]]; then
+	echo "Expected: test time with network policy > test time without network policy."
+	((++pass_fail))
+else
+	echo "Expected: test time with network policy < test time without network policy."
+fi
+
+
+if [[ ( $final_time_np -le 120 ) ]]; then
+	echo "Expected: test time with network policy <= 120 seconds (2 minutes)."
+	((++pass_fail))
+else
+	echo "Expected: test time with network policy >= 120 seconds (2 minutes)."
+fi
+
+
+if [[ ( $final_time_no_np -le 120 ) ]]; then
+	echo "Expected: test time without network policy <= 120 seconds (2 minutes)."
+	((++pass_fail))
+else
+	echo "Expected: test time without network policy >= 120 seconds (2 minutes)."
+fi
+
+
+echo ""
+echo "======Final test result======"
+
+if [[ $pass_fail -eq 3 ]]; then
+	echo -e "\nOverall NetworkPolicy scalability - using customer network policy Testcase result:  PASS"
+  	echo "======Clean up test environment======"
+	# # delete projects:
+  	# ######### Clean up: delete projects and wait until all projects and pods are gone
+  	echo "Deleting test objects"
+	delete_project_by_label $project_label
```

Comment:
with the edits to creating the project above we should use "kube-burner-job" as the label here